docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,72 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+When jobs exceed the memory limits, you will get out of memory (OOM) error and job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits). To avoid OOM error, you can increase the **memory** limits per workflow step to make an efficient use of the available resources. Note that you can also decrease the memory limit to increase the chances of jobs being scheduled earlier if the cluster is busy.
```

Comment:
```suggestion
When jobs exceed the cluster memory limits, you will get out of memory (OOM) error and the job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits). To avoid OOM error, you can increase the **memory** limits per workflow step to make an efficient use of the available resources. Note that you can also decrease the memory limit to increment the chances of jobs being scheduled earlier if the cluster is busy.
```